packages/html-webpack-plugin/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "@talend/html-webpack-plugin",
+  "version": "0.0.1",
+  "description": "@talend/html-webpack-plugin library.",
+  "license": "Apache-2.0",
+  "author": "jmfrancois jmfrancois@talend.com",
+  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "github:Talend/ui"
+  },
+  "devDependencies": {
+    "eslint": "3.4.0",
+    "eslint-config-airbnb": "10.0.1",
+    "eslint-plugin-import": "1.14.0",
+    "jest-cli": "20.0.3",
+    "webpack": "1.13.1"
```

Comment:
`webpack` and `jest` are not useful as dependencies here.

Reply:
I have tests based on jest